src/fake_node_modules/powercord/constants.js

```diff
@@ -2,7 +2,7 @@ const { join } = require('path');
 
 module.exports = Object.freeze({
   // Replugged
-  WEBSITE: 'https://powercord.dev',
+  WEBSITE: 'https://replugged.dev',
```

Comment:
If someone had previously changed this setting and then set it back to `https://powercord.dev`, this change won't apply since their previous setting will override. You should make it change the URL if it's that.

Reply:
Can't really do this just incase somebody wanted to use the powercord api, not sure why they would want to but it's still online and has all the old user stuff on it, chances of somebody actually changing it though is pretty low

Reply:
Wouldn't it not be compatible since you changed stuff?

Reply:
depends if they have api v1 online anymore